packages/generator/src/batch/type.spec.ts

```diff
@@ -13,4 +13,14 @@ describe('type', () => {
     const types = (readRequestType(foodService).type as string).split('|');
     expect(types.join()).not.toMatch(/funcPostReturn/);
   });
+
+  it('considers POST function import for write response', () => {
+    const types = (writeRequestType(foodService).type as string).split('|');
```

Comment:
[q] Wouldn't this work even without the split and join ? Its already typecast to string...